packages/lit-dev-content/samples/examples/context-basics/elements.ts

```diff
@@ -0,0 +1,51 @@
+import {html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {provide, consume, createContext} from '@lit-labs/context';
+
+import {providerStyles} from './styles.js';
+
+const contextKey = Symbol('contextKey');
+// The value type for the context.
+type ContextValue = string;
+// Context object, which acts like a key for the context.
+const context = createContext<ContextValue>(contextKey);
```

Comment:
In a similar vein, let's inline let's inline the `contextKey` constant:

```suggestion
// The context object acts as a key. A consumer will only receive
// values from a provider if their contexts match. A Symbol ensures
// that this context will be unique.
const context = createContext<ContextValue>(Symbol());
```

Reply:
Agree!